rust/signer/src/lib.rs

```diff
@@ -59,16 +59,6 @@ export! {
         qr_reader_phone::decode_sequence(data, cleaned)
     }
 
-    @Java_io_parity_signer_models_SignerDataModel_substratePathCheck
-    fn path_check(
-        seed_name: &str,
-        path: &str,
-        network: &str,
-        dbname: &str
-    ) -> String {
-        db_handling::interface_signer::dynamic_path_check(dbname, seed_name, path, network)
-    }
```

Comment:
It looks like this breaks the iOS build, take a look at the error, check the CI log:

```
❌  Undefined symbols for architecture x86_64
> Symbol: _path_check
> Referenced from: NativeSigner.MDeriveKey.updateDerivationCheck(path: Swift.String, dbName: Swift.String) -> NativeSigner.DerivationCheck? in MDeriveKey.o


❌  ld: symbol(s) not found for architecture x86_64


❌  clang: error: linker command failed with exit code 1 (use -v to see invocation)

** BUILD FAILED **
```

Reply:
Yes, this is result of merge error; should just be reverted